apps/web/pages/signup.tsx

```diff
@@ -161,11 +166,19 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   const ssr = await ssrInit(ctx);
   const token = z.string().optional().parse(ctx.query.token);
 
+  const isEmailVerifyEnabled = await prisma.feature.findFirst({
+    where: {
+      slug: "email-verification",
+      enabled: true,
+    },
+  });
+
   const props = {
     isGoogleLoginEnabled: IS_GOOGLE_LOGIN_ENABLED,
     isSAMLLoginEnabled,
     trpcState: ssr.dehydrate(),
     prepopulateFormValues: undefined,
+    isEmailVerifyEnabled: !!isEmailVerifyEnabled,
   };
```

Comment:
<img width="1171" alt="Screenshot 2023-05-30 at 4 37 11 PM" src="https://github.com/calcom/cal.com/assets/1780212/3047dd90-1470-4c6d-b7b2-1c82a2bc000d">
I see some old verificationToken related code in getServerSideProps. Is that still valid?

Reply:
I believe this is for team invites off the top of my head

Reply:
Yeah it is ^